backend/coreapp/migrations/0017_rename_psyq43_compiler.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations
+import django.db.migrations.operations.special
+
+
+def rename_psyq43(apps, schema_editor):
+    """
+    Restore forks' projects
+    """
+    Scratch = apps.get_model("coreapp", "Scratch")
+    for row in Scratch.objects.all():
+        if row.compiler == "psyq4.3":
+            row.compiler = "gcc2.7.2-psyq"
+        row.save()  # type: ignore
```

Comment:
Bump row.save up into the if (no need to save unless we change the value), remove #type: ignore (my mistake to put it in the previous migration). 

Also you should supply change_fields like we do in the latest version of 0016. Pull master into this to see what I mean

Reply:
I thought I might only need to save the changed rows, but `0013_scratch_name_default` was saving every row, and so I just copied that :)